### ISSUE TITLE
Double discounts not working

### ISSUE THREAD

**Issue by minecraftvillager** [MarioFinale/VillagerSaver#3]
hey, i wanted this plugin so i could turn a villager into a zombie, cure it and get discounts even in easy mode. Normally you can turn the villager into a zombie and cure him twice to get even better discounts. In this plugin even if u cure the zombie ten times the villager will only discount you once. I would be really grateful if you could fix this issue. Thank you so much for the useful plugin.
TL;DR: If you cure a villager twice you get double discounts but with this plugin you only get 1 discount no matter how many times you cured the villagers.

**Comment by MarioFinale** [comment]
Thanks for the report! 
Fixed in [1.1.1](https://github.com/MarioFinale/VillagerSaver/releases/tag/1.1.1)
